snapcraft.yaml

```
@@ -12,18 +12,12 @@ grade: stable
 base: core18
 confinement: strict
 
-environment:
-  USE_qt5: 1
-
 apps:
   nymea-app:
     command: desktop-launch nymea-app
```

You want this for `--shell` environment to be complete:

```suggestion
    command-chain: [desktop-launch]
    command: nymea-app
```

See https://forum.snapcraft.io/t/proposal-support-command-chain-in-apps-and-hooks/6112